typings/svg-properties.d.ts

```diff
@@ -1,308 +1,8 @@
-  radius?: number | string
-  refX?: number | string
-  refY?: number | string
-  renderingIntent?: number | string
-  repeatCount?: number | string
-  repeatDur?: number | string
-  requiredExtensions?: number | string
-  requiredFeatures?: number | string
-  restart?: number | string
-  result?: string
-  rotate?: number | string
-  rx?: number | string
-  ry?: number | string
-  scale?: number | string
-  seed?: number | string
-  shapeRendering?: number | string
-  slope?: number | string
-  spacing?: number | string
-  specularConstant?: number | string
-  specularExponent?: number | string
-  speed?: number | string
-  spreadMethod?: string
-  startOffset?: number | string
-  stdDeviation?: number | string
-  stemh?: number | string
-  stemv?: number | string
-  stitchTiles?: number | string
-  stopColor?: string
-  stopOpacity?: number | string
-  strikethroughPosition?: number | string
-  strikethroughThickness?: number | string
-  string?: number | string
-  stroke?: string
-  strokeDasharray?: string | number
-  strokeDashoffset?: string | number
-  strokeLinecap?: 'butt' | 'round' | 'square' | 'inherit'
-  strokeLinejoin?: 'miter' | 'round' | 'bevel' | 'inherit'
-  strokeMiterlimit?: string
-  strokeOpacity?: number | string
-  strokeWidth?: number | string
-  surfaceScale?: number | string
-  systemLanguage?: number | string
-  tableValues?: number | string
-  targetX?: number | string
-  targetY?: number | string
-  textAnchor?: string
-  textDecoration?: number | string
-  textLength?: number | string
-  textRendering?: number | string
-  to?: number | string
-  transform?: string
-  u1?: number | string
-  u2?: number | string
-  underlinePosition?: number | string
-  underlineThickness?: number | string
-  unicode?: number | string
-  unicodeBidi?: number | string
-  unicodeRange?: number | string
-  unitsPerEm?: number | string
-  vAlphabetic?: number | string
-  values?: string
-  vectorEffect?: number | string
-  version?: string
-  vertAdvY?: number | string
-  vertOriginX?: number | string
-  vertOriginY?: number | string
-  vHanging?: number | string
-  vIdeographic?: number | string
-  viewBox?: string
-  viewTarget?: number | string
-  visibility?: number | string
-  vMathematical?: number | string
-  widths?: number | string
-  wordSpacing?: number | string
-  writingMode?: number | string
-  x1?: number | string
-  x2?: number | string
-  x?: number | string
-  xChannelSelector?: string
-  xHeight?: number | string
-  xlinkActuate?: string
-  xlinkArcrole?: string
-  xlinkHref?: string
-  xlinkRole?: string
-  xlinkShow?: string
-  xlinkTitle?: string
-  xlinkType?: string
-  xmlBase?: string
-  xmlLang?: string
-  xmlns?: string
-  xmlnsXlink?: string
-  xmlSpace?: string
-  y1?: number | string
-  y2?: number | string
-  y?: number | string
-  yChannelSelector?: string
-  z?: number | string
-  zoomAndPan?: string
-}
+export interface SVGPropertiesCompleteSingle
+  extends CSS.SvgProperties<number | string> {}
```

Comment:
should this be `CSS.SvgPropertiesFallback`?
It seems like it would need to be in order to allow using arrays to set fallback values.

Reply:
It does look like it was not supporting array until now, though i'm pretty sure that glamor supports it...

Reply:
I would have used `CSS.SvgPropertiesFallback` if I were writing these typings from scratch, but to minimize breakage I was trying to keep as many of the existing exports intact as possible. So this is accomplishing what `CSS.SvgPropertiesFallback` would:

```ts
type SVGPropertiesComplete = SingleOrArray<
  SVGPropertiesCompleteSingle,
  keyof SVGPropertiesCompleteSingle
>
```